en/docs/tutorials/scenarios/scenario11-graphql.md

```diff
@@ -1,4 +1,42 @@
 # Scenario 11 - GraphQL Support
 
-!!! Note
-    Content to be added.
\ No newline at end of file
+## User Story
+
+**Quantis is more focused on providing more capability to the developer community. They expect the developer community to build their own mobile applications and web apps to use their APIs. To make this process much easier, Quantis wants to expose GraphQL API to the public.**
+
+**_Time to Complete : 5 mins_**
+
+![GraphQL description]({{base_path}}/assets/img/tutorials/scenarios/graphql-schema-import.png)
+
+WSO2 API Manager supports creating GraphQL APIs using the GraphQL schema. Following steps can be used to create a sample API.
+
+1. Log in to publisher portal [https://localhost:9443/publisher](https://localhost:9443/publisher) using apiprovider@quantis.com and password _user123_
```

Comment:
```suggestion
1. Log in to Publisher Portal [https://localhost:9443/publisher](https://localhost:9443/publisher) using apiprovider@quantis.com and password _user123_
```

Let's use Title case for Developer Portal, Publisher Portal, Gateway, Choreo Connect, etc. Please fix in all instances of this.

Reply:
Fixed in all the docs